docs/guides/mustache.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Mustache template with tailcall
+---
+
+Whenever you have a string literal used in a directive, it should support mustache template.The mustache templates serve as placeholders, allowing you to insert dynamic content into your configurations. Simply enclose the dynamic content within double curly braces ({{ }}) wherever you need it within the directive.
+Learn more about mustache template [here](https://mustache.github.io/)
+
+### Leveraging mustache template
+
+#### Contextual Transformation
+
+To get a list of todos you will run the query `todo` defined below.
+
+> [!NOTE]
+> The base url used for this example is https://jsonplaceholder.typicode.com
+
+```graphql
+type Query {
+  todos : [Todo] @http("/todos")
```

Comment:
```suggestion
  todos : [Todo] @http(path: "/todos")
```

Verify if the implemented schema actually works.

Reply:
Sorry, it was a typo from my side